schema/tables/grade_objects.sql

```diff
@@ -22,5 +22,8 @@ CREATE TABLE IF NOT EXISTS grade_objects
     created_date TIMESTAMP,
     tool_name TEXT,
     associated_tool_item_id BIGINT,
-    last_modified TIMESTAMP
+    last_modified TIMESTAMP,
+    short_name TEXT,
+    grade_object_type_id BIGINT,
+    sort_order BIGINT
```

Comment:
Should the docs say they use a column size of 4?

https://documentation.brightspace.com/EN/insights/data_hub/admin/bds_grade_objects.htm?tocpath=Administrators%7CBrightspace%20Analytics%7CData%20Hub%20tool%7CBrightspace%20Data%20Sets%7CGrades%20and%20Grade%20book%7C_____1

(Also just noticed - ShortName is listed as an Integer).